README.md

```diff
@@ -1 +1,197 @@
-# T.B.C
+# DDD Familiarity Assessment
+
+The DDD Familiarity Assessment goal is to provide you with a set of questions to assess the DDD knowledge within your organization.
+
+:warning: DDD Familiarity Assessment __does not__ intend to be a Maturity Model, and it should not be used as such.
+
+Instead, we expect to help organizations identify DDD knowledge gaps and use the results to prepare learnings, training, or similar to fill those gaps.
+
+## Familiarity Assessment Survey
+
+TODO
+
+- How do you assess yourself on DDD? (choose one)
+  - ( ) Never used
+  - ( ) I'm familiar with the concepts
+  - ( ) I'm familiar
+  - ( ) Practitioner
+  - ( ) Expert. Used in several scenarios, failed and learned from the mistakes, and being able to teach others.
+
+- How do you assess yourself on Strategic Design of DDD? (choose one)
+  - ( ) Never used
+  - ( ) I'm familiar with the concepts
+  - ( ) I'm familiar
+  - ( ) Practitioner
+  - ( ) Expert. Used in several scenarios, failed and learned from the mistakes, and being able to teach others.
+
+- How do you assess yourself on Tactical Design of DDD? (choose one)
+  - ( ) Never used
+  - ( ) I'm familiar with the concepts
+  - ( ) I'm familiar
+  - ( ) Practitioner
+  - ( ) Expert. Used in several scenarios, failed and learned from the mistakes, and being able to teach others.
+
+- Which sources have you used to learn about Domain-Driven Design? (Multiple choices)
+  - [ ] Book: Domain-Driven Design by Erik Evans
+  - [ ] Book: Implementing Domain-Driven Design by Vaughn Vernon
+  - [ ] Book: Domain-Driven Design Distilled by Vaughn Vernon
+  - [ ] Book: Learning Domain-Driven Design: Aligning Software Architecture and Business Strategy by Vladik Khononov
+  - [ ] Book: Patterns, Principles, and Practices of Domain-Driven Design by Scott Millett
+  - [ ] Book: Strategic Monoliths and Microservices by Vaughn Vernon
+  - [ ] Conferences
+  - [ ] Blogs and Online Resources
+  - [ ] Workshops and Trainings
+  - [ ] Others... (open text)
+
+### General DDD Concepts
+
+- Which is the most important thing about DDD? It's too open in purpose, there's no right and wrong answer. (Open Text)
+
+- What's a Domain Model? (Open Text)
+
+- What's Domain Modelling? (Open Text)
+
+- In which scenario you would use DDD? (Open Text)
```

Comment:
I've always the found question of when to use and not use DDD is a bit of a false-binary. DDD is a big bag of principles, practices, and patterns that apply to discovery, design, architecture,  and implementing code. You can pick bits and pieces, rather than doing all or nothing.